src/plugins/discord/testUtil.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+export const testChannel = (id: number) => ({
+  id: id,
+  name: "testChannelName",
+  type: 0,
+});
+
+export const testChannelObj = (id: number) => ({
+  id: id,
+  name: "testChannelName",
+  type: "GUILD_TEXT",
+});
+
+export const testMessage = (id: number) => ({
+  id: id,
+  author: {id: 2},
+  timestamp: "2020-03-03T23:35:10.615000+00:00",
+  content: "Just going to drop this here",
+  reactions: [{emoji: {id: 1, name: "testemoji"}}],
+  mentions: [{id: 4, username: "testuser"}],
+});
+
+export const testMember = (userId: number) => ({
+  user: {
+    id: userId,
+    username: "username",
+    discriminator: "disc",
+    bot: true,
+  },
+  nick: "nickname",
+  roles: ["test role"],
+});
```

Comment:
Only one test file is currently using this. So I think it would be better to have this in `fetcher.test.js`. It will make the test file easier to read.

Conventionally such helper functions or example data go at the top of the file unless they are very large snapshots or generic utils.

Reply:
Updated; was originally using this in multiple files, but good suggestion. I move them into the smallest required scope.